rosidl_generator_py/package.xml

```diff
@@ -10,13 +10,28 @@
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend>rosidl_generator_c</buildtool_export_depend>
 
+  <build_depend>rmw</build_depend>
+  <build_depend>rmw_implementation</build_depend>
+  <build_depend>rosidl_generator_c</build_depend>
```

Comment:
I've moved the build dependencies to the top, though I've always seen the buildtool_depend first and then the other dependencies. Does this look ok? https://github.com/ros2/rosidl/commit/b9cf5eb36dab4075951f7f4a8bc3575db9079dc7